lib/store_web/emails/authemail.ex

```diff
@@ -0,0 +1,17 @@
+defmodule StoreWeb.Emails.AuthEmails do
+  @moduledoc """
+  A module to build auth related emails.
+  """
+  use Phoenix.Swoosh, view: StoreWeb.EmailView, layout: {StoreWeb.LayoutView, :email}
+
+  def reset_password_email(id, to: email) do
+    frontend_url = Application.fetch_env!(:bokken, StoreWeb.Endpoint)[:frontend_url]
+
+    new()
+    |> from({"CeSIUM", "noreply@store.cesium.di.uminho.pt"})
```

Comment:
```suggestion
    |> from({"CeSIUM - Store", "noreply@store.cesium.di.uminho.pt"})
```

You forgot this one 💪🏼